src/sqlitestorage.cpp

```diff
@@ -1039,8 +900,30 @@ bool SqliteStorage::insertedIncidences(Incidence::List *list, const QDateTime &a
             qsize1 = sizeof(SELECT_COMPONENTS_BY_CREATED);
         }
 
-        return d->selectIncidences(list, query1, qsize1,
-                                   DBInsert, after, notebookUid);
+        qCDebug(lcMkcal) << "incidences inserted since" << after;
+        if (!d->mSem.acquire()) {
+            qCWarning(lcMkcal) << "cannot lock" << d->mDatabaseName << "error" << d->mSem.errorString();
+            return false;
+        }
+
+        SL3_prepare_v2(d->mDatabase, query1, qsize1, &stmt1, nullptr);
+        secs = d->mFormat->toOriginTime(after);
+        SL3_bind_int64(stmt1, index, secs);
+        if (!notebookUid.isNull()) {
```

Comment:
Was already like that, but think isEmpty() could be more correct than isNull. Same thing on many other places here.

Reply:
Sure, changed in all the query methods.